modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/pages/FormBuilder.es.js

```diff
@@ -86,6 +86,18 @@ export const FormBuilder = () => {
 
 	const addToast = useToast();
 
+	useEffect(() => {
+		const sessionLength = Liferay.Session ? Liferay.Session.get('sessionLength') : 60000;
```

Comment:
I did some tests and realized that every time I reload the form builder page, the Liferay.Session is null in the useEffect, that's why I made these changes compared to the implementation in the Page Editor.